tests/kafkatest/services/kafka/templates/log4j.properties

```diff
@@ -20,86 +20,98 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
 
 # INFO level appenders
-log4j.appender.kafkaInfoAppender=org.apache.log4j.DailyRollingFileAppender
-log4j.appender.kafkaInfoAppender.DatePattern='.'yyyy-MM-dd-HH
+log4j.appender.kafkaInfoAppender=org.apache.log4j.RollingFileAppender
```

Comment:
tests log4j.properties don't need to change

Reply:
Alright, I'll make a change.